src/System.Device.Gpio/System/Device/Gpio/Drivers/SysFsDriver.Linux.cs

```diff
@@ -66,6 +72,36 @@ public SysFsDriver()
             _eventThreadCancellationTokenSource = new CancellationTokenSource();
         }
 
+        /// <summary>
+        /// The internal timeout for the event wait function.
+        /// </summary>
+        internal TimeSpan PollingTimeout
+        {
+            get
+            {
+                return TimeSpan.FromMilliseconds(_pollingTimeoutInMilliseconds);
```

Comment:
If we intend to do this conversion everytime, why not just sotre it as a TimeSpan instead?